src/forms/button/Button.tsx

```diff
@@ -44,10 +45,14 @@ export enum ButtonSize {
 // TODO: add loading support
 
 export class Button extends React.PureComponent<ButtonProps> {
+    static defaultProps = {
+        customBtn: true,
```

Comment:
I think 'customBtn' should be false initially. As you need to customize the Button for  your special requirement.

Reply:
@vinodwani18, I'm still seeing this defaulted to `true`.

Reply:
As discussed with Arpita changed button name from customBtn to defaultBtn